ebean-core/pom.xml

```diff
@@ -72,7 +72,7 @@
     <dependency>
       <groupId>io.ebean</groupId>
       <artifactId>ebean-ddl-generator</artifactId>
-      <version>12.9.0</version>
+      <version>12.9.3a</version>
```

Comment:
There are cyclic dependencies. I build it with skipTests and  then executed the maven command
```
mvn install:install-file -DgroupId=io.ebean -DartifactId=ebean-ddl-generator -Dpackaging=jar -Dversion=12.9.3a -Dfile=ebean-ddl-generator/target/ebean-ddl-generator-12.9.3-SNAPSHOT.jar
```
to put it in my local repo

Reply:
Ah, I think it can be just a prior version of ebean-ddl-generator.  I don't think we want 3a here.

Reply:
Putting this back to 12.9.2 (previously released ebean-ddl-generator).  At some point these tests that need ddl generation (most of them) need to move to break the cycle.